frontend/lib/filesize/index.ts

```diff
@@ -0,0 +1,9 @@
+export const formatFileSize = (bytes: number): string => {
+  if (bytes === 0) return 'Empty'
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB']
+
+  const k = 1024
```

Comment:
Apologies for the hideous pedantry, especially since you aren't technically wrong here, but I'd use `kB` if you're doing base-10, and `KiB` if you're doing base-2 [1].

When talking about file sizes, one normally uses the base 10 variant, e.g. [this random SSD](https://www.samsung.com/us/computing/memory-storage/solid-state-drives/990-pro-w-heatsink-pcie-4-0-nvme-ssd-4tb-mz-v9p4t0cw/) has 4 trillion bytes

Obligatory xkcd: https://xkcd.com/394/

[1] https://cseducators.stackexchange.com/a/4431

Reply:
Not Pedantry! Sounds correct :)